packages/retail-ui-extensions/src/globals.ts

```diff
@@ -0,0 +1,15 @@
+import type {ExtensionPoints} from './extension-points';
+
+export interface ShopifyGlobal {
+  extend<ExtensionPoint extends keyof ExtensionPoints>(
+    extensionPoint: ExtensionPoint,
+    extend: ExtensionPoints[ExtensionPoint],
+  ): void;
+  reload(): void;
+}
+
+declare global {
+  interface WorkerGlobalScope {
+    readonly le_shopify: ShopifyGlobal;
```

Comment:
I think you want `shopify` here, checkout had to do this for <reasons> that you probably don't need to worry about